### ISSUE TITLE
Story on "Deploy cf using bosh + cf-deployment" should give the pair a starting point

### ISSUE THREAD

**Issue by kquan-work** [pivotal/cf-onboarding#225]
https://www.pivotaltracker.com/story/show/163527200
under 'How?' 
"Clone the cf-deployment repo to your workstation, then follow _these instructions_ (you've already done the stemcell step, so just double-check that the stemcell versions still line up)."

---> recommend updating the description to advise the pair to start at the 'upload a runtime-config'  section in the github. 

Also, the instructions "We recommended that you use the one provided by the bosh-deployment repo..." 
---> recommend calling out to the pair they should clone the bosh-deployment repo before running the `bosh update` command

**Comment by dirkjot** [comment]
I also had many issues with this story (going through onboarding myself right now): 
- the story points to `cf-deployment`  instructions, which contain 8 steps
- because of the way those steps are written (do this, but if using `bbl`, do that) this is hard to follow
- because we have already done a fair few things, almost all steps do not apply.
- the instructions are not very clear about also cloning `bosh-deployment` and the command provided for that is incorrect (you have to be inside the bosh deployment directory, which changes the paths)
- it is not clear to me what we do with the bosh-deployment

**Comment by thausler786** [comment]
@dirkjot @kquan-work Alright, the fixes I am going to make so far are:

- including the cloning of `bosh-deployment` in the Tracker story so you already have it when you enter the instructions
- set expectations in the story that the pair won't have to do all of the steps because you've already done most of them.

@dirkjot I was a bit curious if you could provide extra guidance around what was confusing about the use of bosh deployment...I'm assuming that it's step 6 in the [deployment guide](https://github.com/cloudfoundry/cf-deployment/blob/master/deployment-guide.md) that you are talking about.

@heyjcollins ^ tagging you for feedback on your deployment guide

**Comment by dirkjot** [comment]
My plan was to write a story with all the steps in it , referring to the
document only for further reference. I obviously haven't gotten around to
that yet.

Your edits are great.

**Comment by thausler786** [comment]
Closing this issue, glad to accept more clarifying edits if people get around to it.  Thanks everyone!
